webapp/src/components/modals/channel_subscriptions/edit_channel_subscription.tsx

```diff
@@ -66,10 +66,13 @@ export type State = {
     jiraIssueMetadata: IssueMetadata | null;
     error: string | null;
     getMetaDataErr: string | null;
+    getIssueStatusesErr: string | null;
     submitting: boolean;
     subscriptionName: string | null;
     showConfirmModal: boolean;
     conflictingError: string | null;
+    issue_statuses: Status[] | null;
+    projectStatuses: ProjectStatuses[] | null;
```

Comment:
The mix of snake case and camel case seems weird here. All of the other keys are camel case so I'm thinking we should go with that